examples/diagram-fit-on-load/README.md

```diff
@@ -28,5 +28,8 @@ const bpmnVisualization = new BpmnVisualization(window.document.getElementById(c
 
 const bpmnContent = ``; // your BPMN 2.0 XML content
 const fitTypeValue = 'Center'; // From radio button or select
```

Comment:
:warning: same remark as in the other fit example, let advice people to use the constant instead of raw string + simplify the code